tests/framework/base/ModelTest.php

```diff
@@ -462,10 +462,21 @@ public function testValidateWriteOnly()
 
         $this->assertTrue($model->validate());
     }
+
+    public function testValidateAttributeNames()
+    {
+        $model = new ComplexModel1();
+        $model->name = 'Some value';
+        $this->assertTrue($model->validate(['name']), 'Should validate only name attribute');
+        $this->assertTrue($model->validate('name'), 'Should validate only name attribute');
+        $this->assertFalse($model->validate(), 'Should validate all attributes');
+    }
 }
 
 class ComplexModel1 extends Model
 {
+    public $name, $description, $id, $is_disabled;
```

Comment:
Maybe place every property at new line for [PSR-2 compability](http://www.php-fig.org/psr/psr-2/#42-properties)?